tools/themetool.bat

```diff
@@ -61,58 +61,66 @@ set /a i=%i%+1
 convert -rotate 90 "%~nx1" bgr:"%~n1.bin"
 GOTO:EOF
 :makepng-all
+set i=0
+if not exist "TOP.bin" (
+	echo Cannot find "TOP.bin"! Aborted.
+	goto:EOF
+)
+if not exist "cfg0TOP.bin" (
+	echo Cannot find "cfg0TOP.bin"! Aborted.
+	goto:EOF
+)
 ren TOP.bin TOP.bi_
 ren cfg0TOP.bin cfg0TOP.bi_
-for /r %%f in (*.bin) do (
-call :topng %%f 320
+for /r %%f in (*.bin) do call :topng %%f 320
```

Comment:
Why search for TOP.bi_ and cfg0TOP.bin_?
Also, I think it should continue even without hero-new-3ds.png.
Well, I'll merge it for now...